### PR TITLE
cli: Add archiving option for storage-add subcommand

### PR DESCRIPTION
**Pull request by vatsa287**
This PR adds new option '--pvReclaimPolicy" to storage-add subcommand
enabling PVC archiving support.

Updates: #535
Signed-off-by: Shree Vatsa N <vatsa@kadalu.io>

### CODE REVIEW

**Review by leelavg — APPROVED** [review]
Might be good to add o/p of a dry-run? like `# python __main__.py storage-add ..`

File: cli/README.md [hunk 1]
```diff
@@ -61,6 +61,16 @@ $ kubectl kadalu storage-add storage-pool-3 \
     --pvc azure-disk-1
 ```
 
+To `archive` persistent volume claims,
+> Note:
+> PV Reclaim Policy is set to `delete` by default
+
+```
+$ kubectl kadalu storage-add storage-pool-1 \
+    --pvReclaimPolicy=archive
```
Comment (aravindavk) [inline cli/README.md]: For uniformity, please change to `--pv-reclaim-policy`

**Comment by vatsa287** [comment]
--pvReclaimPolicy=archive
```
vatsa@vatsa-VirtualBox:~/kadalu/cli$ sudo build/kubectl-kadalu storage-add --device=vatsa-virtualbox:/mnt/disk1 storage-pool-1 --pvReclaimPolicy=archive
The following nodes are available:
  vatsa-virtualbox

Storage Yaml file for your reference:

apiVersion: "kadalu-operator.storage/v1alpha1"
kind: "KadaluStorage"
metadata:
  name: "storage-pool-1"
spec:
  type: "Replica1"
  storage:
    - node: "vatsa-virtualbox"
      device: "/mnt/disk1"
  pvReclaimPolicy: archive

Is this correct?(Yes/No): 
```
--pvReclaimPolicy=delete

```
vatsa@vatsa-VirtualBox:~/kadalu/cli$ sudo build/kubectl-kadalu storage-add --device=vatsa-virtualbox:/mnt/disk1 storage-pool-1 --pvReclaimPolicy=delete
The following nodes are available:
  vatsa-virtualbox

Storage Yaml file for your reference:

apiVersion: "kadalu-operator.storage/v1alpha1"
kind: "KadaluStorage"
metadata:
  name: "storage-pool-1"
spec:
  type: "Replica1"
  storage:
    - node: "vatsa-virtualbox"
      device: "/mnt/disk1"
  pvReclaimPolicy: delete

Is this correct?(Yes/No): 
```
Option not used
```
vatsa@vatsa-VirtualBox:~/kadalu/cli$ sudo build/kubectl-kadalu storage-add --device=vatsa-virtualbox:/mnt/disk1 storage-pool-1 
The following nodes are available:
  vatsa-virtualbox

Storage Yaml file for your reference:

apiVersion: "kadalu-operator.storage/v1alpha1"
kind: "KadaluStorage"
metadata:
  name: "storage-pool-1"
spec:
  type: "Replica1"
  storage:
    - node: "vatsa-virtualbox"
      device: "/mnt/disk1"

Is this correct?(Yes/No): 
```

File: cli/README.md (hunk 1 above)
Comment (vatsa287) [reply cli/README.md]: sure

Comment (vatsa287) [reply cli/README.md]: --pv-reclaim-policy
```
vatsa@vatsa-VirtualBox:~/kadalu/cli$ sudo build/kubectl-kadalu storage-add --device=vatsa-virtualbox:/mnt/disk1 storage-pool-1 --pv-reclaim-policy=archive
The following nodes are available:
  vatsa-virtualbox

Storage Yaml file for your reference:

apiVersion: "kadalu-operator.storage/v1alpha1"
kind: "KadaluStorage"
metadata:
  name: "storage-pool-1"
spec:
  type: "Replica1"
  storage:
    - node: "vatsa-virtualbox"
      device: "/mnt/disk1"
  pvReclaimPolicy: archive

Is this correct?(Yes/No): 
```

Comment (aravindavk) [inline cli/README.md]: doc is still unchanged

Comment (vatsa287) [reply cli/README.md]: done

**Review by aravindavk — APPROVED** [review]

**Pull request merged**